test/test_roscpp/test/src/params.cpp

```diff
@@ -528,6 +528,38 @@ TEST(Params, mapBoolParam)
   ASSERT_TRUE(std::equal(map_b.begin(), map_b.end(), map_b2.begin()));
 }
 
+TEST(Params, paramTemplateFunction)
+{
+  EXPECT_EQ( param::param<std::string>( "string", "" ), "test" );
+  EXPECT_EQ( param::param<std::string>( "gnirts", "test" ), "test" );
+
+  EXPECT_TRUE( param::param<int>( "int", 0 ) == 10 );
```

Comment:
Could you make the EXPECT_EQ change here and for every integer test too? Thanks!